modules/network-observability-enriched-flows.adoc

```diff
@@ -6,10 +6,11 @@
 [id="network-observability-enriched-flows_{context}"]
 = Export enriched network flow data
 
-You can send network flows to Kafka, IPFIX, or both at the same time. Any processor or storage that supports Kafka or IPFIX input, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data.
+You can send network flows to Kafka, IPFIX, the Red Hat build of OpenTelemetry, or all three at the same time. For Kafka or IPFIX, any processor or storage that supports those inputs, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data. For OpenTelemetry, network flow data and metrics can be exported to a compatible OpenTelemetry endpoint, such as Red Hat build of OpenTelemetry, Jaeger, or Prometheus. 
 
 .Prerequisites
-* Your Kafka or IPFIX collector endpoint(s) are available from Network Observability `flowlogs-pipeline` pods.
+* Your Kafka, IPFIX, or OpenTelemetry collector endpoint(s) are available from Network Observability `flowlogs-pipeline` pods.
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses#plurals) we should avoid using (s) to indicate that something is potentially plural. In this case, I feel like making it plural makes sense:
```suggestion
* Your Kafka, IPFIX, or OpenTelemetry collector endpoints are available from Network Observability `flowlogs-pipeline` pods.
```

But totally up to you if you think another option is better